soccerdata/fbref.py

```diff
@@ -193,7 +216,8 @@ def read_seasons(self) -> pd.DataFrame:
         df.drop_duplicates(subset=["league", "season"], keep="first", inplace=True)
         df = df.set_index(["league", "season"]).sort_index()
         return df.loc[
-            df.index.isin(list(itertools.product(self.leagues, self.seasons))), ["format", "url"]
+            df.index.isin(list(itertools.product(df_leagues.index.unique(), self.seasons))),
+            ["format", "url"],
```

Comment:
This is equivalent to: `df.loc[(slice(None), self.seasons) , ["format", "url"]]`.